src/main/java/com/booster/server/CreateNoteInput.java

```diff
@@ -0,0 +1,12 @@
+package com.booster.server;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@Getter
+@Setter
+@Accessors(chain = true)
+public class CreateNoteInput {
+    String content;
```

Comment:
Please add a `private` modifier and a `@ToString` here so we could see the content in the logs.
Please do the same for the `CreateVocabularyEntryInput`

Reply:
Done